src/app/CliTools/Console/Command/TYPO3/BeUserCommand.php

```diff
@@ -197,48 +197,6 @@ protected function setTypo3UserForDatabase($database, $username, $password)
         );
         $tsConfig = implode("\n", $tsConfig);
 
-        // Default uc
-        $uc = array(
-            'thumbnailsByDefault' => 1,
-            'recursiveDelete' => 1,
-            'showHiddenFilesAndFolders' => 1,
-            'edit_RTE' => 1,
-            'resizeTextareas' => 1,
-            'resizeTextareas_Flexible' => 1,
-            'copyLevels' => 99,
```

Comment:
@denislorch, schade, dass das rausgeflogen ist, fand das immer gut, dass die copyLevels und recursiveDelete für den `dev` User schon gesetzt waren.

Reply:
Das ist aber hier die falsche Stelle. Man sollte im Projekt UserTS in der `*core` Extension haben für diese Defaults, und diese für Admins z.B. aktivieren. Vor allen wenn wir in Zukunft ggfl nicht nur einen "cron" User hätten wäre das hilfreich. Das greift dann natürlich auch im Docker und kann z.B. auch andere sinnvolle Defaults enthalten.

Reply:
@baschny , hm, hab das jetzt 3 Wochen probiert, aber manches fehlt mir doch. Grad zum Beispiel `uc/moduleData/web_ts/ts_browser_type=setup`. Glaub das kannst du nicht über UserTS setzen. Wie könnte man das deiner Meinung nach sauber integrieren?

Reply:
Nevermind, hab's ;-)

```
setup.default.moduleData.web_ts.ts_browser_type = setup
```